tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -733,7 +733,7 @@ def test_ht003_kgofailure(self):
                                   timeout_sec=self.msg_timeout)
 
             # Run a rolling restart.
-            self.stage_rolling_restart()
```

Comment:
`test_ht003_kgofailure()` is the only method that uses `stage_rolling_restart()`. so, i've replaced `stage_rolling_restart()` with the service level `rolling_restart_pods()`